README.md

```diff
@@ -1,11 +1,15 @@
 # Project Name
 
-Replace this readme with your own information about your project. Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The Pizzeria-bot assignment is an introduction to JavaScrips and meant to challenge and teach me how to use alerts (pop-up windows), write clean JS code, the use of variables and the use of data types - among many other new things regarding JS. The Assignment was divided into 5 steps with some sub-steps.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+After looking through all the provided material by Technigo I started by writing down each step and the requirement for each step on a post-it note each, to then lay them out on a row. This way I could focus on each post-it (step) without getting overwhelmed or being distracted by the other steps. When done, I flipped the post-it note and went on with the next post-it (step).
+
+Step 1 and 2 felt pretty straight forward and simple, while step 3 got me into a position of thiking and scrathicng my head. After looking though different options of solving the problem I started trying out. I tried different approaches in CodePen until I was satisfied and copy/pasted it into VScode. The hardest part was figuring out how I initally wanted it to look, and how to make the code work, but when I finally uderstood it, the rest felt simple.
+
+If I would have more time on my hands, I ould have worked more woth understanding everything, there are still many things that I don't understand. I wouls also make the code loop when incorrect answers were given, until the right one is entered.
 
```

Comment:
I really enjoyed reading about your process in the README as I found it to be quite interesting and relatable, but there are some typos which made it a little distracting.